### PR TITLE
[MIG] donation: Migration to 13.0

### PR DESCRIPTION
**Pull request by albariera**
Standard migration

Depends on #85

### CODE REVIEW

File: donation/models/donation.py [hunk 1]
```diff
@@ -25,29 +25,30 @@ class DonationDonation(models.Model):
...
-            )
-            tax_receipt_total_cc = donation_currency._convert(
-                tax_receipt_total,
-                company_currency,
-                donation.company_id,
-                donation.donation_date,
-            )
-            donation.amount_total_company_currency = total_company_currency
-            donation.tax_receipt_total = tax_receipt_total_cc
+            if donation.donation_date:
```
Comment (etobella) [inline donation/models/donation.py:28]: Maybe we need and else, WDYT?

**Review by benbrich — APPROVED** [review]
Functional Review Legit!

File: donation/models/donation.py (hunk 1 above)
Comment (albariera) [reply donation/models/donation.py:28]: You are right! Thank you :)

**Review by etobella — APPROVED** [review]

**Pull request merged**